package/cfg/rke2-cis-1.23-hardened/master.yaml

```diff
@@ -96,6 +96,38 @@ groups:
           For example, chown root:root $schedulerconf
         scored: true
 
+      - id: 1.1.7
+        text: "Ensure that the etcd pod specification file permissions are set to 600 or more restrictive (Automated)"
+        audit: "/bin/sh -c 'if test -e $etcdconf; then find $etcdconf -name '*etcd*' | xargs stat -c permissions=%a; fi'"
+        use_multiple_values: true
```

Comment:
```suggestion
        audit: "/bin/sh -c 'if test -e $etcdconf; then stat -c permissions=%a $etcdconf; fi'"
```
IIRC There is only one pod manifest/config in RKE2 `/var/lib/rancher/k3s/server/db/etcd/config` so we don't need to use this `mutiple_values` approach  and the `find $etcdconf -name '*etcd*' `  seems wrong, as will not return anything.
<img width="722" alt="image" src="https://github.com/user-attachments/assets/6f1b21e1-693e-490c-9269-d8c5efc98901">